ltr_scripts/train.sh

```diff
@@ -4,7 +4,7 @@ OUTPUT_DIR=${OUTPUT_DIR:-../tmp/libsvm}
 TRAIN=${TRAIN:-../tmp/ltr_data/train.txt}
 VALI=${VALI:-../tmp/ltr_data/validate.txt}
 TEST=${TEST:-../tmp/ltr_data/test.txt}
-STEPS=${STEPS:-50000} # runs at ~1,000 rounds per minute. 50,000 steps = 50 mins.
+STEPS=${STEPS:-100000} # runs at ~1,000 rounds per minute. 50,000 steps = 50 mins.
```

Comment:
Should this change have been committed? (and if it should can the comment be updated?)